schemas/oic.sec.ace2.json

```diff
@@ -1,11 +1,18 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.sec.ace2.json#",
-  "title": "Subject based Access Control Entry (ACE) object definition",
+  "title": "Subject-based Access Control Entry (ACE) object definition",
   "definitions": {
     "oic.sec.ace2": {
       "type": "object",
       "properties": {
+        "aceid": {
```

Comment:
We've made "doxm.sct" and "doxm.oxms" mandatory in the schema but they are not included in UPDATE request messages.  We need to be consistent with how we define "aceid" which is to make it mandatory.  This is what every other WG does with their schema files and it's what we did with several other SVR Properties.